src/server/v2.0/handler/preheat.go

```diff
@@ -80,3 +89,149 @@ func convertProvidersToFrontend(backend []*provider.Metadata) (frontend []*model
 	}
 	return
 }
+
+// GetPolicy is Get a preheat policy
+func (api *preheatAPI) GetPolicy(ctx context.Context, params operation.GetPolicyParams) middleware.Responder {
+	project, err := api.projectCtl.GetByName(ctx, params.ProjectName)
+	if err != nil {
+		return api.SendError(ctx, err)
+	}
+
+	var payload *models.PreheatPolicy
+	policy, err := api.preheatCtl.GetPolicyByName(ctx, project.ProjectID, params.PreheatPolicyName)
+	if err != nil {
+		return api.SendError(ctx, err)
+	}
+
+	payload, err = convertPolicyToPayload(policy)
+	if err != nil {
+		return api.SendError(ctx, err)
+	}
+	return operation.NewGetPolicyOK().WithPayload(payload)
+}
+
+// CreatePolicy is Create a preheat policy under a project
+func (api *preheatAPI) CreatePolicy(ctx context.Context, params operation.CreatePolicyParams) middleware.Responder {
```

Comment:
Two things we need to make sure here as well as the `UpdatePolicy` method:
  
- name uniqueness checking 
- the name should be limited to the specified format which should be aligned with the URL pattern ([\d_]+)?